src/test/groovy/com/stormpath/tck/authentication/CookieIT.groovy

```diff
@@ -91,54 +73,43 @@ class CookieIT extends AbstractIT {
         assertTrue(isCookieDeleted(response.detailedCookies.get("refresh_token")))
     }
 
-    /** Reject unauthorized text/html requests with 302 to login route
-     * 302 to login route should preserve original URL including query string
-     * @see <a href="https://github.com/stormpath/stormpath-framework-tck/issues/36">#36</a>
-     * @see <a href="https://github.com/stormpath/stormpath-framework-tck/issues/121">#121</a>
-     * @throws Exception
-     */
-    @Test(groups=["v100", "html"])
-    public void unauthorizedHtmlRequestIsForwardedToLogin() throws Exception {
```

Comment:
I don't think @omgitstom ever weighed in on this. It doesn't make sense (to me) in the context of a JSON *only* endpoint to redirect to an HTML endpoint.